src/LoginScreen.js

```diff
@@ -1,9 +1,61 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {loginWithUserData} from './helperFunctions/login';
+import {updateJWT} from './actions';
+import './stylesheets/login.css';
 
-let LoginScreen = () => 
-    <div>
-        <p>Login Screen</p>
-    </div>
 
 
+let mapStateToProps = (state) => {
+    return {jwt: state.jwt}
+};  
+
+let mapDispatchToProps = (dispatch) => {
+  return {dispatch: dispatch}
+};
+
+class LoginScreenDumb extends Component {
```

Comment:
Making a class is unnecessary here since we have a reducer.  Instead of a class, you can set LoginScreenDumb to the JSX info you have in your render area.  All helper functions can all be separate as well.  Your LoginScreenDub will take as arguments the deconstructed { jwt, dispatch }.    I'll write some of the other changes in future comments.